app/models/mixins/miq_provision_mixin.rb

```diff
@@ -70,9 +70,9 @@ def get_owner
   end
 
   def set_owner_group(user)
```

Comment:
Since you're changing the method, would it be more efficient as:
```ruby
def set_owner_group(user)
  return unless (group = user.miq_groups.where(:description => get_option(:owner_group)).first)
  user.current_group = group
end
```